src/interface/report/Results/EncounterStats.tsx

```diff
@@ -114,9 +114,7 @@ class EncounterStats extends PureComponent<Props, State> {
 
     return fetchWcl<WCLRankingsResponse>(`rankings/encounter/${this.props.currentBoss}`, {
       // TODO once we have completed migrating to v2, remove the class/spec params
-      class: this.props.config.spec.ranking.class,
-      spec: this.props.config.spec.ranking.spec,
-      className: this.props.config.spec.wclClassName,
+      className: this.props.config.spec.wclClassName.replace(' ', ''),
```

Comment:
didn't mean to include this here but I cba putting it in another PR, so here we are